test/modules/layers/geojson-layer.spec.js

```diff
@@ -126,7 +128,107 @@ test('GeoJsonLayer#tests', t => {
     },
     props: {
       // TODO: Set a right geojson example as the provided from 'deck.gl-data' contains 'GeometryCollection' types that are not compatible with geojsonToBinary
-      data: geojsonToBinary(binaryData)
+      data: binaryData
+    }
+  });
+
+  const binaryDataWithAttributes = {
+    ...binaryData,
+    points: {
+      ...binaryData.points,
+      attributes: {
+        getRadius: {
+          size: 1,
+          value: binaryData.points.featureIds.value.map(_ => Math.round(Math.random() * 10))
```

Comment:
I would avoid having random behavior in tests as errors can then be unreproducible. If you want a range of values you could always do `(_, i) => 10 *i`

Reply:
Fixed.